proto/zitadel/resources/action/v3alpha/action_service.proto

```diff
@@ -135,7 +135,7 @@ service ZITADELActions {
           description: "Target successfully created";
           schema: {
             json_schema: {
-              ref: "#/definitions/CreateTargetResponse";
+              ref: "#/definitions/v3alphaCreateTargetResponse";
```

Comment:
is this change intended?

Reply:
Yes, the generate throws errors otherwise, was v2CreateTargetResponse before, which worked, then was CreateTargetResponse, so I changed it to v3alpha...

Reply:
> correct refs to messages in proto which are used for doc generation

Does this comment refer to this change?
What does that do? I'm just curious.

Reply:
![image](https://github.com/user-attachments/assets/73786cba-21e0-40c7-a6fc-f0378f950ec2)

Reply:
Thats for the status code 201, otherwise it would only be done generated in the swagger for 200.

Reply:
thanks